app/components/ProductCard/ProductCard.js

```diff
@@ -19,30 +20,46 @@ export const DefaultCardWrapper = styled.div`
   background-color: ${({ color }) => color};
 `;
 
-export const AvatarWrapper = styled.div`
+export const Avatar = styled.div`
   display: flex;
   justify-content: center;
-  height: 80%;
-`;
-
-export const Avatar = styled.img`
-  height: 300px;
+  height: 70%;
+  background-image: url(${({ avatarUrl }) => avatarUrl});
+  background-color: #f6f6f6;
+  background-position: 50% 35%;
+  background-size: 155%;
+  background-repeat: no-repeat;
+  will-change: background-image;
+  will-change: background-size;
```

Comment:
`will-change: background-image, background-size;`

Otherwise you are overwriting

Reply:
Styled components?

Reply:
Great catch! Thanks!